src/engine/shared/config_variables.h

```diff
@@ -55,6 +55,7 @@ MACRO_CONFIG_INT(InfAccusationThreshold, inf_accusation_threshold, 4, 0, 8, CFGF
 MACRO_CONFIG_INT(InfLeaverBanTime, inf_leaver_ban_time, 5, 0, 180, CFGFLAG_SERVER, "How long an infected gets banned (in minutes), when leaving and leaving causes a human to get infected")
 MACRO_CONFIG_INT(InfFastDownload, inf_fast_download, 1, 0, 1, CFGFLAG_SERVER, "Enables fast download of maps")
 MACRO_CONFIG_INT(InfMapWindow, inf_map_window, 15, 0, 100, CFGFLAG_SERVER, "Map downloading send-ahead window")
+MACRO_CONFIG_INT(InfMinRoundsForMapVote, inf_min_rounds_for_map_vote, 1, 0, 100, CFGFLAG_SERVER, "Minimum number of rounds before a new map can be voted")
```

Comment:
Please set default to 0 to maintain current behavior. Server admins will set it to values most suitable for their servers.